Userland/Games/Minesweeper/Field.cpp

```diff
@@ -278,14 +324,14 @@ void Field::reset()
             });
             square.number = number;
             if (square.has_mine)
-                continue;
-            if (square.number)
+                square.label->set_icon(m_mine_bitmap);
+            else if (square.number) {
                 square.label->set_icon(m_number_bitmap[square.number - 1]);
+            }
```

Comment:
These changes seem unrelated to the rest of this commit, unless I'm missing something.

Reply:
This one was fixing a bug introduced when the generation was changed, would it be better if the fix was in a separate commit?